demos/mixer-config-quota-bookinfo.yaml

```diff
@@ -42,17 +77,13 @@ data:
             # we want to increment this counter by 1 for each unique (source, target, service, method, response_code) tuple
             value: "1"
             labels:
-              source: source.service | "unknown"
+              source: source.name | "unknown"
```

Comment:
I think you either want both "source.name" and "target.name" or "source.service" and "target.service".